README-source.md

```diff
@@ -1339,8 +1348,8 @@ Paging is **only used when a trigger is part of a dynamic dropdown**. Depending
 
 Paging is a lot like a regular trigger except the range of items returned is dynamic. The most common example of this is when you can pass a `start` parameter:
```

Comment:
"when you can pass a `start` parameter:" 
-> 
"when you can pass an `offset` parameter:"
?

I know this isn't something you changed in this PR, but it's odd we say `start` and then use `offset` just below in the example